scripts/healthcheck.js

```diff
@@ -0,0 +1,29 @@
+const dotenv = require('dotenv')
+dotenv.config({ path: `${__dirname}/../.env` })
+const { exec } = require('child_process');
+
+const webhookUrl = process.env.HEALTHCHECK_WEBHOOK
+
+function sendToDiscord() {
+    fetch(webhookUrl, {
+        method: 'post',
+        body:    JSON.stringify({ content: '@dev No successful notifications in the last 24 hours' }),
```

Comment:
@dev won't work, one moment, I'll get you proper tag

Reply:
Use this one: https://github.com/BigWhaleLabs/ketl-discord-backend/blob/main/src/models/DiscordRoles.ts